ArmPkg/Include/Library/NotificationServiceLib.h

```diff
@@ -0,0 +1,96 @@
+/** @file
+  Definitions for the Notification Service
+
+  Copyright (c), Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+
+**/
+
+#ifndef NOTIFICATION_SERVICE_LIB_H_
+#define NOTIFICATION_SERVICE_LIB_H_
+
+#include <Base.h>
+#include <IndustryStandard/ArmFfaPartInfo.h>
+#include <Library/ArmSvcLib.h>
+#include <Library/ArmFfaLibEx.h>

```

Comment:
nit: we generally do not recommend carrying header files in the header files. But we can worry about it later.